src/codeu/chat/server/Snapshotter.java

```diff
@@ -0,0 +1,57 @@
+package codeu.chat.server;
+// Where should this file and StoredData be saved?
+//  In the same folder as Model.java?
+
+import codeu.chat.server.Model;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+
+public class Snapshotter implements Runnable {
+
+    private Model model;
+
+    public Snapshotter(Model model) {
+        this.model = model;
+    }
+
+    public void run() {
+        boolean infinite = true; 
+        while (infinite = true) { // I feel like this isnt a good way to make an infinite while loop but
+            sleep();
+            serialize();
+        }
+        
+    }
+
+    public void serialize() {
+        try {
+        // StoredData.txt is a file to save the serialized data to
+        FileOutputStream fs = new FileOutputStream("StoredData.txt");
```

Comment:
You should be using the same file path to `serialize` the data as you do the `deserialize`. Pass in the `fp` from `Server` into the constructor of snapshotter and use that file in `FileOutputStream`.